forge-gui/res/cardsfolder/s/synth_eradicator.txt

```diff
@@ -0,0 +1,15 @@
+Name:Synth Eradicator
+ManaCost:2 R
+Types:Artifact Creature Synth Soldier
+PT:3/3
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME attacks, exile the top card of your library. You may get {E}{E} (two energy counters). If you don't, you may play that card this turn.
+SVar:TrigExile:DB$ Dig | Defined$ You | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBChoice
+SVar:DBChoice:DB$ GenericChoice | Choices$ DBEnergy,DBPlay
+SVar:DBEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 2 | SubAbility$ DBCleanup | SpellDescription$ You get 2 energy counters
```

Comment:
RememberPut$ + Condition after should be better

Reply:
Hmm this isn't working now

Reply:
somehow I don't think you will Remember a "Card" :P

Reply:
Hmm this new version isn't working as well. Revert to previous version ?

Reply:
I'll debug

Reply:
your `Remembered$Amount` is only for cards...
imo just replacing "Card" with "Player" should have worked 💩

Reply:
thanks working now